backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/description/ToolDescriptionService.java

```diff
@@ -190,4 +192,40 @@ public NodeTool createNodeToolFromDiagramBackground(NodeDescription nodeDescript
                 .build();
     }
 
+    public NodeTool createNodeToolFromDiagramWithDirection(NodeDescription nodeDescription, EClass eClass, FeatureDirectionKind direction, IDescriptionNameGenerator descriptionNameGenerator) {
```

Comment:
Is this method the same than the others in this PR?
If yes, could you share the code in a service class please?

Reply:
Done